dataprofiler/profilers/profile_builder.py

```diff
@@ -683,3 +687,127 @@ def tqdm(l):
             
         return profile
 
+    def _delete_data_labelers(self):
+        """
+        Helper method for deleting all data labelers before saving to disk.
+        """
+        # Delete data labeler for profiler
+        data_labeler_options = self.options.structured_options.data_labeler
+        if data_labeler_options.is_enabled \
+                and data_labeler_options.data_labeler_object is not None:
+                    data_labeler_options.data_labeler_object = None
+                
+        # Delete data labelers for all columns
+        for key in self._profile:
+
+            val = self._profile[key]
+            use_data_labeler = True
+            if val.options and isinstance(val.options, StructuredOptions):
+                use_data_labeler = val.options.data_labeler.is_enabled
+
+            if use_data_labeler:
+                val.profiles['data_label_profile']._profiles['data_labeler'] \
+                   .data_labeler = None
+
+    def _restore_data_labelers(self):
+        """
+        Helper method for restoring all data labelers after saving to or 
+        loading from disk.
+        """
+        # Restore data labeler for profiler
+        data_labeler_options = self.options.structured_options.data_labeler
+        if data_labeler_options.is_enabled \
+                and data_labeler_options.data_labeler_object is None:
+            try:
+                data_labeler = DataLabeler(
+                    labeler_type='structured',
+                    dirpath=data_labeler_options.data_labeler_dirpath,
+                    load_options=None)
+                self.options.set({'data_labeler.data_labeler_object': data_labeler})
```

Comment:
Isn't there a name or something we have to load these (aka we can match with)? @JGSweets

Reply:
I'm not sure what you mean. 

in the old version before we updated, it used to be `name` was associated to the labeler in our our repo, but that was since removed I believe. Best we can do now is serialize the DL being used (recognize if lib DL and store / reload it with that knowledge)

Reply:
actually, we can load the processors/models that way. And if we send in the info to the DL it should be able to be loaded with just names if it is within the lib

Reply:
`BaseDataLabeler.load_from_library(name=...)`

Reply:
So we can save the DL name if it is from our own lib. And on reload use this.